apps/cowswap-frontend/src/modules/application/containers/App/index.tsx

```diff
@@ -147,6 +147,7 @@ export function App() {
         <RedirectAnySwapAffectedUsers />
         <ThemeProvider />
         <GlobalStyles />
+        <ScrollToHashElement />
```

Comment:
Changed!

Replaced `react-router-hash-link` with `@cascadia-code/scroll-to-hash-element`.
This is the setup for it.